lib/random_data.rb

```diff
@@ -0,0 +1,21 @@
+module RandomData
```

Comment:
My guess is that this `RandomData` module should either:

- `require 'faker'` and use `Faker::…` to implement `random_paragraph` and `random_sentence`
- be dropped entirely and you'd keep using `Faker::…` in db/seeds.rb as it pretty much gets the job done already